spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -2052,14 +2085,15 @@ def stub_requested_attributes
     end
   end
 
+  # temporarily commenting out this spec because it needs to be updated to work
```

Comment:
**question:** I'm guessing this was meant to be deleted?

Reply:
Yep! Fixed in https://github.com/18F/identity-idp/pull/7616/commits/114280bea2a9b129da53a143317111a7f234fbdd